services/app-web/src/pages/MccrsId/MccrsId.tsx

```diff
@@ -71,7 +133,9 @@ export const MccrsId = (): React.ReactElement => {
             const updateResult = await updateFormData({
                 variables: {
                     input: {
-                        mccrsID: values?.mccrsId?.toString().replace(/ /g, ''),
+                        mccrsID:
+                            values?.mccrsId?.toString().replace(/ /g, '') ||
+                            undefined,
```

Comment:
Passes `undefined` to the database since the delete flow will not be implemented